.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
can you add `include-hidden-files: true` under `if-no-files-found: ignore`? this will sort the coverage issue

Reply:
also `PYTHON_LATEST: "3.12"` -> `PYTHON_LATEST: "3.13"`

Reply:
Pipeline is completing now. Thanks for that.